test/extensions/filters/http/cors/cors_filter_test.cc

```diff
@@ -758,6 +759,72 @@ TEST_F(CorsFilterTest, OptionsRequestWithWildcardAllowHeaders) {
   EXPECT_EQ(Http::FilterTrailersStatus::Continue, filter_.encodeTrailers(response_trailers_));
 }
 
+TEST_F(CorsFilterTest, OptionsRequestMatchingOriginByWildcardWithPNA) {
+  Http::TestRequestHeaderMapImpl request_headers{
+      {":method", "OPTIONS"},
+      {"origin", "test-host"},
+      {"access-control-request-method", "GET"},
+      {"access-control-request-private-network", "true"}};
```

Comment:
May we test a additional case that the request has no `access-control-request-private-network` header or the `access-control-request-private-network` header with invalid value?